ui/src/menu.tsx

```diff
@@ -31,17 +31,19 @@ const css = stylesheet({
 export interface Menu {
   /** Commands to render. */
   items: Command[]
-  /** The card's icon. Mutually exclusive with the image. */
+  /** The card's icon. Mutually exclusive with the image and label. */
   icon?: S
-  /** The card’s image, preferably user avatar. Mutually exclusive with the icon. */
+  /** The card’s image, preferably user avatar. Mutually exclusive with the icon and label. */
   image?: S
   /** An identifying name for this component. */
   name?: Id
+  /** The card's label. Mutually exclusive with the icon and image */
+  label?: S
```

Comment:
Let's call this title instead of label.
```suggestion
  title?: S
```